person.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'nameable'
 
 class Person < Nameable
-  attr_reader :id
+  attr_reader :id, rentals
   attr_accessor :name, :age
```

Comment:
- [ ] Kindly consider adding the `rentals` in the attr_accessor and dont forget to prefixing rentals with color `:` to make the syntax right.